python/fluidity/state_types.py

```diff
@@ -20,8 +20,8 @@ def __init__(self,n=""):
   def __repr__(self):
     return '(State) %s' % self.name
   def print_fields(self):
-    print "scalar: ",self.scalar_fields,"\nvector:",self.vector_fields,"\ntensor:", \
-    self.tensor_fields,"\ncsr_matrices:", self.csr_matrices
+    print("scalar: ",self.scalar_fields,"\nvector:",self.vector_fields,"\ntensor:", \
```

Comment:
This actually prints something else now in python2.7:
```
$ python -c 'print("scalar: ", 3, "\nvector:", 4)'
('scalar: ', 3, '\nvector:', 4)
$ python3 -c 'print("scalar: ", 3, "\nvector:", 4)'
scalar:  3 
vector: 4
```
so maybe add a `from __future__ import print_function` on top? Only an issue for print statements with more than one argument...